### ISSUE TITLE
Make TS Syntax Errors more clear

### ISSUE THREAD

**Issue by f3d0r** [dotansimha/graphql-code-generator#3453]
When there is a TS build error, we get the following message upon running `graphql-codegen`:
```
  Parse configuration
  Generate outputs
    Generate packages/[omitted]
      Load GraphQL schemas
      Load GraphQL documents
        Identifier 'Text' has already been declared (14:7)
        Generate
    Generate packages/[omitted]
      Load GraphQL schemas
      Load GraphQL documents
      Identifier 'Text' has already been declared (14:7)
        Generate
```

This is a TS error, but since building our packages relies on the correct generated GraphQL types already existing, the GraphQL type/codegen has to be run first, so it is first to catch TS compile errors.

It's hard for us to find the file where this error is occurring without more information. Can the codegen output the file path where this TS compile error exists? i.e. `Identifier 'Text' has already been declared ([file-path]:14:7)`

Thank you!

**Comment by adelsz** [comment]
@f3d0r This is fixed in v1.13.1
Kudos to this PR https://github.com/ardatan/graphql-toolkit/pull/477

**Comment by f3d0r** [comment]
Thank you!
